src/telemetry/telemetry.ts

```diff
@@ -140,6 +140,15 @@ class Telemetry implements ITelemetry {
       ''
     );
   }
+
+  public getUserId(){
```

Comment:
This function hasn't been defined in the interface

Reply:
Ah nice catch let me add that